src/app/(public)/public.css

```
@@ -0,0 +1,1151 @@
+.public-shell h1,
+.public-shell h2,
+.public-shell h3 {
+  font-weight: 700;
+}
+
+.public-visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)